mutalyzer/GenRecord.py

```diff
@@ -312,6 +312,21 @@ def findGene(self, name) :
         return None
     #findGene
 
+    def getInfoByTranscriptID(self, transcript_id):
```

Comment:
Actually the name *index* has disappeared in my current code, I now always work with the complete ``GENE_v001`` string which I call *name* (which on LRG references would be ``t1``). You could still think of it as being composed of a gene symbol and transcript index in the RefSeq case of course.

Anyway, naming things is complex and for sure Mutalyzer has a lot of confusing names at the moment, so I wouldn't worry about it too much :smile: